server/README.md

```diff
@@ -0,0 +1,321 @@
+# BraveSensor-Server
+
+[![Build Status](https://travis-ci.com/bravetechnologycoop/BraveSensor-Server.svg?branch=master)](https://travis-ci.com/bravetechnologycoop/BraveSensor-Server)
+
+# How to release a new version of BraveSensor-Server
+
+## 1. Update git to reflect the new release
+
+1. on your local machine, in the `BraveSensor-Server` repository:
+
+   1. pull the latest code ready for release: `git checkout devenv && git pull origin devenv`
+
+   1. decide on an appropriate version number for the new version
```

Comment:
This is going to be interesting since what will be a breaking change in firmware won't be in server, and vice versa. Maybe we need to start using two tags? Like a `server-v5.0.0` and `firmware-v1.3.0`?

Reply:
is this true? I think breaking change still has a system-wide definition, can you think of an example of one of these changes that "breaks" in firmware and not in server?